### PR TITLE
Restricted channel source to webshop #7560

### PR DESCRIPTION
**Pull request by rajatguptarg**

### CODE REVIEW

File: website.py [hunk 1]
```diff
@@ -37,7 +37,10 @@ class Website:
     #:
     channel = fields.Many2One(
         'sale.channel', 'Channel', required=True,
-        domain=[('create_users', '=', Eval('application_user'))],
+        domain=[
+            ('create_users', '=', Eval('application_user')),
+            ('source', '=', 'webshop'),
+        ],
         depends=['application_user']
     )
```
Comment (tarunbhardwaj) [inline website.py]: add in source https://github.com/openlabs/trytond-amazon-mws/blob/develop/channel.py#L81-L91

File: channel.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+"""
+    channle.py
+    :copyright: (c) 2015 by Openlabs Technologies & Consulting (P) Limited
```
Comment (tarunbhardwaj) [inline channel.py]: one line space

File: channel.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+"""
+    channle.py
```
Comment (tarunbhardwaj) [inline channel.py]: typo

File: channel.py [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+__all__ = ['SaleChannel']
+
+
+class SaleChannel:
+    """
+    Sale Channel
+    """
+    __name__ = 'sale.channel'
+
+    # Override the get_source method to add 'Webshop' as a source in channel
```
Comment (tarunbhardwaj) [inline channel.py]: Comment should be inside `def`

File: website.py (hunk 1 above)
Comment (rajatguptarg) [inline website.py]: Done

**Pull request merged**